Plugin/Source/PluginProcessor.cpp

```diff
@@ -134,10 +134,15 @@ float ChowtapeModelAudioProcessor::calcLatencySamples() const noexcept
 
 bool ChowtapeModelAudioProcessor::isBusesLayoutSupported (const BusesLayout& layouts) const
 {
-    return ((! layouts.getMainInputChannelSet().isDiscreteLayout())
-            && (! layouts.getMainOutputChannelSet().isDiscreteLayout())
-            && (layouts.getMainInputChannelSet() == layouts.getMainOutputChannelSet())
-            && (! layouts.getMainInputChannelSet().isDisabled()));
+    if (wrapperType == juce::AudioProcessor::WrapperType::wrapperType_LV2) {
+        return ((layouts.getMainOutputChannelSet() == AudioChannelSet::stereo())
+                && (layouts.getMainInputChannelSet() == AudioChannelSet::stereo()));
+    } else {
+        return ((! layouts.getMainInputChannelSet().isDiscreteLayout())
+                && (! layouts.getMainOutputChannelSet().isDiscreteLayout())
+                && (layouts.getMainInputChannelSet() == layouts.getMainOutputChannelSet())
+                && (layouts.getMainOutputChannelSet() == AudioChannelSet::stereo()));
```

Comment:
```suggestion
                && (! layouts.getMainInputChannelSet().isDisabled()));
```
Seems like this last check is not equivalent to what we had here before?

Reply:
Sorry, I really don't know how I managed to mess this up. I restored this check to the exact form in was before.

Reply:
All good! That's what the CI is for :)